src/pages/Game/components/PlanetIcon/index.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import './styles.scss'
+import saturn from '../../../../assets/images/saturn.png'
+import earth from '../../../../assets/images/earth.png'
+
+const PlanetIcon = ({player}) => {
+  /*colocar estilo en styles*/
```

Comment:
Never leave personal comments in the code. If you need to communicate the placement of a future implementation use a `TODO: ...` comment, and always thinking that you are communicating this to other coworkers (and always in english).